service/server.go

```diff
@@ -337,9 +337,9 @@ func (s *server) listenAndServeAPI() error {
 }
 
 func (s *server) healthzSrvEnable() bool {
```

Comment:
Since the `HealthzPort` has been changed, I think this name should be changed as well.
What about `healthCheckSrvEnable()`, etc.

Reply:
fixed in: 41e9d6b
use short form `hcSrvEnable` for private methods